glib/src/boxed_any_object.rs

```diff
@@ -0,0 +1,172 @@
+// Take a look at the license at the top of the repository in the LICENSE file.
+
+use crate as glib;
```

Comment:
Unrelated to this merge request, but please open an issue regarding the limitation /bug of glib-macros